### PR TITLE
Portfolio Site

### PR DESCRIPTION
**Pull request by MoyasiGinko**
### In this milestone project:

- created a new GitHub repo "Portfolio-Site".
- created a new branch "mainpage-setup-files".
- created a ".gitignore" file.
- added "node_modules/" to ".gitignore" file.
- created ".github/workflows/linters.yml" file.
- added license.md file and modified the README.md file.
- added dependencies for html & css.
- added index.html file and style.css file.
- created a portfolio site main page using figma template.

### CODE REVIEW

**Review by amon-cofie — CHANGES_REQUESTED** [review]
Hi @MoyasiGinko , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You don't have all the required sections of the readme. Please note that all sections of the readme template, with the exception of the FAQ section, are required and must be included in your readme file. Sections that do not apply to your project can be marked as ```not applicable``` 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @MoyasiGinko , 



Indeed you did an exceptional but still, there is room for changes to complete this project. 💯

### Hats off to you for copying the design exceptionally well.


## Highlights 
✅Design is really really well implemented.
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                </div>
+                <div>
+                    <p>
+                        I'm a software developer! I can help you build a product , feature or website Look through some
+                        of my work and experience!
+                        If you like what you see and have a project you need coded, don't hestiate to contact me.
+                    </p>
+                </div>
+                <div id="social__container">
+                    <p id="social-heading">Let's Connect</p>
```
[the remark this answers is not in the document]
Comment (MoyasiGinko) [reply index.html]: i have changed the tag to "a tag" with href="#". hope that is right?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                    <span class="stake-bottom"></span>
+                </button>
+            </nav>
+        </header>
+        <section class="main-container">
+            <div class="main-content-intro">
+                <div>
+                    <h1>I'm Alexandra. <br />
+                        Glad to see you!
+                    </h1>
```
[the remark this answers is not in the document]
Comment (MoyasiGinko) [reply index.html]: I have put my name. Thank you for your suggestion.

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @MoyasiGinko  

Your project is complete! Great job for making the changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request✔️ 
- figma design ok✔️ 
- guidelines well followed ✔️ 
- No linter errors✔️ 

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**